### PR TITLE
see detail device model tooltip changed

### PR DESCRIPTION
**Pull request by ssgueye2**
## Description
Change the detail buttton tooltip for device model into `see device model details`
What's new?

- #1995 

## What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Tests
- [x] Other

### CODE REVIEW

File: .github/workflows/publish-documentation-new-version.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ jobs:
     name: Publish Documentation with New Version
     runs-on: ubuntu-latest
     steps:
+      - name: Set env.BRANCH
+        run: echo "BRANCH=$(echo $GITHUB_REF | cut -d'/' -f 3)" >> $GITHUB_ENV
```
Comment (hocinehacherouf) [inline .github/workflows/publish-documentation-new-version.yml:11]: Why did you add this step ?

Comment (ssgueye2) [reply .github/workflows/publish-documentation-new-version.yml:11]: I did not add it. I just did a rebase in origin/main-vnext. I suppose that it was added by kevin in main-vnnext branch

Comment (hocinehacherouf) [reply .github/workflows/publish-documentation-new-version.yml]: I dropped the commit ok kevin on your branch

**Review by hocinehacherouf — APPROVED** [review]
